message_sender/tasks.py

```diff
@@ -111,6 +111,13 @@ def run(self, message_id, **kwargs):
                 # send or resend
                 try:
                     if "voice_speech_url" in message.metadata:
+
+                        # OBD number of tries metric
+                        fire_metric.apply_async(kwargs={
+                            "metric_name": 'vumimessage.obd.tries.sum',
+                            "metric_value": 1.0
+                        })
```

Comment:
The metric description indicates that it id for OBD(Outbound dialer) only, Does this include text messages?